_docs/concepts/security/rbac.md

```diff
@@ -0,0 +1,221 @@
+
+## Architecture
+
+The diagram below shows Istio RBAC architecture. The admins specify Istio RBAC policies. The policies are saved in Istio config store.
+
+{% include figure.html width='80%' ratio='56.25%'
+    img='./img/IstioRBAC.svg'
+    alt='Istio RBAC'
+    title='Istio RBAC Architecture'
+    caption='Istio RBAC Architecture'
+    %}
+
+Istio RBAC engine does two things:
+* **Fetch RBAC policy.** Istio RBAC engine watches for changes on RBAC policy. It fetches the updated RBAC policy if it sees any changes.
+* **Authorize Requests.** At runtime, when a request comes, the request context is passed to Istio RBAC engine. RBAC engine evaluates the
+request context against the RBAC policies, and returns the authorization result (ALLOW or DENY).
+
+### Request Context
+
+In the current release, Istio RBAC engine is implemented as a [Mixer adapter]({{home}}/docs/concepts/policy-and-control/mixer.html#adapters).
+The request context is provided as an instance of the
+[authorization template](https://github.com/istio/istio/blob/master/mixer/template/authorization/template.proto). The request context
+ contains all the information about the request and the environment that an authorization module needs to know. In particular, it has two parts:
+
+* **subject** contains a list of properties that identify the caller identity, including "user" name/ID, "groups" the subject belongs to,
+or any additional properties about the subject such as namespace, service name.
+* **action** specifies "how a service is accessed". It includes "namespace", "service", "path", "method" that the action is being taken on,
+and any additional properties about the action.
+
+Below we show an example "requestcontext".
+```rule
+   apiVersion: "config.istio.io/v1alpha2"
+   kind: authorization
+   metadata:
+     name: requestcontext
+     namespace: istio-system
+   spec:
+     subject:
+       user: request.auth.principal | ""
+       groups: request.auth.principal | ""
+       properties:
+         service: source.service | ""
+         namespace: source.namespace | ""
+     action:
+       namespace: destination.namespace | ""
+       service: destination.service | ""
+       method: request.method | ""
+       path: request.path | ""
+       properties:
+         version: request.headers["version"] | ""
+```
+
+## Istio RBAC Policy
+
+Istio RBAC introduces ServiceRole and ServiceRoleBinding, both of which are defined as Kubernetes CustomResourceDefinition (CRD) objects.
+
+* **ServiceRole** defines a role for access to services in the mesh.
+* **ServiceRoleBinding** grants a role to subjects (e.g., a user, a group, a service).
+
+### ServiceRole
+
+A ServiceRole specification includes a list of rules. Each rule has the following standard fields:
+* **services**: a list of services.
+* **methods**: HTTP methods or gRPC methods. Note that gRPC methods should be presented in the form of "packageName.serviceName/methodName".
```

Comment:
Are they case sensitive? Worthwhile documenting either way

Reply:
Spike, thanks for the review! Added (case sensitive).